setup.py

```diff
@@ -2,43 +2,57 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
+from __future__ import unicode_literals
 
 import os
 import io
+import json
 from setuptools import setup, find_packages
 
-BASEDIR = os.path.dirname(__file__)
 
+# Helpers
+def read(*paths):
+    """Read a text file."""
+    basedir = os.path.dirname(__file__)
+    fullpath = os.path.join(basedir, *paths)
+    contents = io.open(fullpath, encoding='utf-8').read().strip()
+    return contents
 
-def read(f, method='read'):
-    """Read a text file at the given relative path."""
-    filepath = os.path.join(BASEDIR, f)
-    return getattr(io.open(filepath, encoding='utf-8'), method)().strip()
 
+# Prepare
 PACKAGE = 'oki'
-VERSION = read(os.path.join(BASEDIR, PACKAGE, 'VERSION'), 'readline')
+INSTALL_REQUIRES = [
+    'six>=1.9',
+]
+LINT_REQUIRES = [
```

Comment:
@roll we can and should lint with tests. if there are too many errors (on Tabulator I guess you mean, as here there are not), we need to change the pylint config, or the code :).

Related, according to pylint docs, `pylintrc` is better than `.pylintrc` (the latter will get overriden by a global pylint config if present)